typify-impl/src/convert.rs

```diff
@@ -2115,6 +2116,35 @@ mod tests {
         validate_builtin!(std::collections::BTreeSet<u32>);
     }
 
+    #[test]
```

Comment:
I'd suggest deleting this test and adding some types to `typify/tests/schemas/simple-types.json`. These are typically easier to create, easier to update, and easier to understand.

Reply:
Hey @ahl Is there an easy way to apply the patch change to the rs file?